### PR TITLE
Add Copyright Owner and Update Year

### PR DESCRIPTION
**Pull request by SteveDeakin**
Added the Copyright Owner (Crown Copyright) and Updated Year to 2016-2017

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```
Comment (chrisns) [inline LICENSE]: Hi,
Thanks for your contribution!
It however needs to be:
`Copyright (c) 2016-2017 Crown`
as per https://opensource.org/licenses/MIT

Thanks!
Chris

Comment (SteveDeakin) [inline LICENSE]: Chris,

No problem.

Annoyingly UK Law refers to it as Crown Copyright http://www.legislation.gov.uk/ukpga/1988/48/section/163

We could change to "Copyright (c) 2016-2016 Crown Copyright" to be really picky.

Steve

Comment (daniel-ac-martin) [inline LICENSE]: If I recall, it's more usual to do this sort of format:
```
Copyright (c) 2010-15,17 Crown copyright
```
i.e. Something that was worked on from 2010 to 2017 in all years but 2016.

Comment (daniel-ac-martin) [inline LICENSE]: Actually, ignore me. Just had a google. Looks like lots of formats are used.

Comment (daniel-ac-martin) [inline LICENSE]: The above is pretty concise though.

Comment (SteveDeakin) [inline LICENSE]: Strictly speaking for US Law - it's first year of publication https://www.copyright.gov/title17/92chap4.html#401 of the original work and last year of modification. So 2010-2015 would mean we claim first copyright from 2010 but the protection is from 2015 onwards (125 year for the Crown in the UK) as it has been modified.

Comment (SteveDeakin) [inline LICENSE]: https://www.copyrightservice.co.uk/copyright/p03_copyright_notices is a useful guide

Comment (SteveDeakin) [inline LICENSE]: This a good view from US side - http://www.photoattorney.com/qa-what-year-should-i-use-for-my-copyright-notice/

Comment (SteveDeakin) [inline LICENSE]: Good plan. My original concern was the Copyright holder.

**Comment by chrisns** [comment]
closed due to inactivity

**Pull request closed without merging**